debezium-core/src/main/java/io/debezium/transforms/strategy/DeleteExtractRecordStrategy.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms.strategy;
+
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Struct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.DebeziumException;
+import io.debezium.transforms.ExtractNewRecordStateConfigDefinition.DeleteHandling;
+
+/**
+ * Deprecated, use {@link DeleteTombstoneExtractRecordStrategy} instead
+ *
+ * @author Harvey Yue
+ */
+@Deprecated
+public class DeleteExtractRecordStrategy<R extends ConnectRecord<R>> extends AbstractExtractRecordStrategy<R> {
```

Comment:
Shouldn't this be a Strategy below the `ExtractNewRecordState` layer and then be named something like `...DeleteHandlingStrategy` instead of `...ExtractRecordStrategy`?
Maybe I am mistaken here though... (see my other comment about the strategy name as well)